templates/download/raspberry-pi/index.html

```diff
@@ -0,0 +1,109 @@
+      <h3 class="p-heading--four u-no-margin--bottom">Raspberry Pi 2</h3>
+    </div>
+    <div class="col-3">
+      <div><img src="https://assets.ubuntu.com/v1/c4a007b6-Raspberry+Pi+3.png" alt=""></div>
+      <h3 class="p-heading--four u-no-margin--bottom">Raspberry Pi 3</h3>
+    </div>
+    <div class="col-3">
+      <div><img src="https://assets.ubuntu.com/v1/a999bb5f-Raspberry+Pi+4.png" alt=""></div>
+      <h3 class="p-heading--four u-no-margin--bottom">Raspberry Pi 4</h3>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered is-shallow">
+  <div class="u-fixed-width">
+    <h4>Ubuntu {{ releases.lts.full_version }} LTS</h4>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-3">
+      <p class="u-no-margin--bottom">The version of Ubuntu with long term support, until {{ releases.lts.eol }}.</p>
+    </div>
+    <div class="col-3">
+      <p class="u-sv2">&nbsp;</p>
+      <p class="u-no-margin--bottom"><a href="/download/raspberry-pi/thank-you?version={{ releases.lts.full_version }}&amp;architecture=armhf+raspi2" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 2 32-bit', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">Download 32-bit</a></p>
+    </div>
+    <div class="col-3">
+      <p><a href="/download/raspberry-pi/thank-you?version={{ releases.lts.full_version }}&amp;architecture=arm64+raspi3" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 3 64-bit', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">Download 64-bit</a></p>
+      <p class="u-no-margin--bottom"><a href="/download/raspberry-pi/thank-you?version={{ releases.lts.full_version }}&amp;architecture=armhf+raspi3" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 3 32-bit', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">Download 32-bit</a></p>
+    </div>
+    <div class="col-3">
+      <p><a href="/download/raspberry-pi/thank-you?version={{ releases.lts.full_version }}&amp;architecture=arm63+raspi3" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 4 64-bit', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">Download 64-bit</a></p>
+      <p class="u-no-margin--bottom"><a href="/download/raspberry-pi/thank-you?version={{ releases.lts.full_version }}&amp;architecture=armhf+raspi3" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 4 32-bit', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">Download 32-bit</a></p>
```

Comment:
Shouldn't these be downloading raspi4?

Reply:
That does not currently exist as a separate image.